### PR TITLE
Reorganize README content

### LINKED ISSUE #1207: Add more information to top-level README
Now the repository is public and so more people new to the project will be first arriving at the top-level repository page https://github.com/UCL/TLOmodel, I think it would be useful to add a bit more detail about the project to the README file that is rendered there - for example description of project and brief overview of model (possibly reusing some of existing text from `docs`), links to documentation website, contributor guidelines.

Currently the `README.rst` is pulled in to the main documentation by

https://github.com/UCL/TLOmodel/blob/f88df3e4903d9806406a24a8a245789940f5553b/docs/readme.rst#L1

which might not make sense if this was being considered as more of a standalone overview rather than specifically 'getting started' instructions as currently.

### PR DESCRIPTION
**Pull request by matt-graham**
Fixes #1207 

Adds a new top-level `README.md` file which includes the TLO logo, a brief project description, badges for CI status + DOI link, links to key documentation pages and other resources, summary of project team and acknowledgement.

Moves existing `README.rst` to `docs/getting_started.rst` as well as removing `docs/readme.rst` which just includes the top-level `README.rst` file and updating reference in `docs/index.rst` table of contents. Also fixes a formatting issue in installation instructions code block introduced in #1307 and removes DOI badge from this page.

Rationale for this change is that as the repository is now public, having a bit more introductory text in the README I think is useful to give people who come across the repository a better idea of TLOmodel is and who develops it. At the moment I haven't included any specific installation / getting started instructions in the README in favour of linking to the documentation page, with the view that this avoids repetition and makes it less likely these will get out of date with each other.

### CODE REVIEW

**Review by tbhallett — APPROVED** [review]
This looks FANTASTIC -- thanks very much @matt-graham

**Comment by tbhallett** [comment]
oh whoops --- I see that you were waiting for a review from @tamuri too. Hope that's all Ok. It looks great to me.

**Comment by matt-graham** [comment]
No problem about not waiting for the review from Asif @tbhallett - I mainly tagged for additional pair of 👀 but as this is only changing docs pages not code, I think one reviewer is fine! Thanks for approving and merging

**Pull request merged**
